src/Core/Kernel.php

```diff
@@ -350,42 +348,12 @@ protected function getCacheHash(): string
         ]);
     }
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:remove - all connection variables are done in MySQLFactory
```

Comment:
all availables reasons can be found here: \Shopware\Core\DevOps\StaticAnalyze\PHPStan\Rules\Deprecation\DeprecatedMethodsThrowDeprecationRule::RULE_EXCEPTIONS

with one of those added to the comment, PHPStan will not complain about missing "Feature::triggerDeprecationOrThrow" call. 

but in this case we should add this call